integrate-kobiton-travisci.md

```diff
@@ -0,0 +1,246 @@
+
+The main steps of this section:
+1. Use Hockey App API to get the latest app version and its download URL.
+2. Setting desired capabilities
+3. Adding test script execution command
+
+### 4.1. Fetching the latest app version and app download URL using Hockey App API
+
+To make things simplier, we have provided a bash script that will handle the first step as well as execute your automation test script.
+
+Click [here](sample-scripts/test-with-kobiton.sh) and copy the file to your automation test folder.
+
+This script will use Hockey App REST API to fetch your latest build version and your app's download link to 2 variables `VERSION` and `RESIGNED_URL`.
+
+### 4.2 Adding test script execution command
+Kobiton already provides some samples for automation test, visit [here](https://github.com/kobiton/samples) for reference.
+
+**In this guideline, we will use the `Node.js` sample (`samples/javascript` folder) as an example.**
+
+Open the `bash` file in the previous step.
+
+At the end of the file, add a line to execute your automation test script based on this format:
+```bash
+VERSION=$version RESIGNED_URL=$resignedUrl <YOUR_AUTOMATION_SCRIPT_EXCEUTION_COMMAND>
+```
+
+Because we're using the Kobiton sample, we will use `android-app-test.js` as our automation test script. Therefore, the execution command in the end of the `bash` file should be like:
+```bash
+VERSION=$version RESIGNED_URL=$resignedUrl npm android-app-test
+```
+
+### 4.3 Setting Desired Capabilities
+We will use Kobiton’s desired capabilities settings to configure your test scripts. Our settings include all the required configurations you need to automate your test using the Appium Framework and to run your automation test on real devices in the Kobiton cloud.
+
+Open automation test script file or create a new one.
+
+Replace `desiredCaps` in your script with ones you got from the previous step like the template below.
+Notice the `app` element will be the value of `RESIGNED_URL` which is the download URL of your app.
+
+```javascript
+var desiredCaps = {
+  sessionName:        'Automation test session',
+  sessionDescription: '', 
+  deviceOrientation:  'portrait',  
+  captureScreenshots: true, 
+  app:                process.env.RESIGNED_URL, 
+  deviceGroup:        'KOBITON', 
+  deviceName:         '<KOBITON_DEVICE_NAME>',
+  platformVersion:    '<KOBITON_DEVICE_VERSION>',
+  platformName:       '<DEVICE_PLATFORM_NAME>' 
+} 
+```
+
+## 5. Configuring Travis CI
+Testing with Kobiton should run dependently with deploying process. Therefore, we will put this section to a different build stage so that Travis CI will execute it only when the deploying stage is completed successfully.
+
+Open `.travis.yml` file in your project
+
+We will assume you already configured a `Deploy` stage for archiving Reat Native project and uploading the latest app version to Hockey App.
```

Comment:
**have already configured** **React**.
This line is confusing, please explain it in an easy-to-understand way :D